addon/globalPlugins/ocr.py

```diff
@@ -257,25 +289,56 @@ class OCRSettingsPanel(gui.SettingsPanel):
 
 	def makeSettings(self, settingsSizer):
 		sHelper = gui.guiHelper.BoxSizerHelper(self, sizer = settingsSizer)
+		# Translators: Label of a radiobox used to choose a quality recognition
+		recogQualityLabel = _("During OCR, prefer")
+		self.recogQualityRB = sHelper.addItem(wx.RadioBox(self, label=recogQualityLabel, choices=[x[0] for x in OCR_QUALITIES.values()]))
+		quality = config.conf["ocr"]["quality"]
+		select = [k for k,v in OCR_QUALITIES.items() if v[1] == quality][0]
+		self.recogQualityRB.SetSelection(select)
+		self.recogQualityRB.Bind(wx.EVT_RADIOBOX, self.onQualityChange)
 		# Translators: Label of a  combobox used to choose a recognition language
 		recogLanguageLabel = _("Recognition &language")
 		self.recogLanguageCB = sHelper.addLabeledControl(
 			recogLanguageLabel,
 			wx.Choice,
-			choices=[lang.localizedName for lang in LanguageInfo.fromAvailableLanguages()],
+			choices=[],
 			style = wx.CB_SORT
 		)
+		self.updateCB()
+		# Translators: Label of a  combobox used to choose a recognition priority
+		recogPriorityLabel = _("Recognition &priority")
+		self.recogPriorityCB = sHelper.addLabeledControl(
+			recogPriorityLabel,
+			wx.Choice,
+			choices=[x[0] for x in OCR_PRIORITIES.values()]
+		)
+		priorityID = config.conf["ocr"]["priority"]
+		select = self.recogPriorityCB.FindString(OCR_PRIORITIES[priorityID][0])
+		self.recogPriorityCB.SetSelection(select)
+
+	def onQualityChange(self, evt):
+		select = self.recogQualityRB.GetSelection()
+		config.conf["ocr"]["quality"] = OCR_QUALITIES[select][1]

```

Comment:
You shouldn't save anything into the config unless user presses okay. In the current situation if user changes quality and then closes the dialog without saving his choice is remembered in the config.